backend/index.js

```diff
@@ -1,5 +1,6 @@
 // Import the Express module
-const express = require('express');
+import express from 'express';
+import { products } from './constants.js';
```

Comment:
this is un-needed we are going to just pull from mongo, I don't want un-needed files in the project cuz we will forget to delete.

Reply:
Could this be removed once mongo is successfully integrated? Someone may be using the dummy data for some frontend issues